### ISSUE TITLE
Provide feedback of ISO currently being used by another process sooner in Edit-OSDCloudWinPE

### ISSUE THREAD

**Issue by tekuadam** [OSDeploy/OSD#191]
**Describe the solution you'd like**
If OSDCloud.iso / OSDCloud_NoPrompt.iso are currently being used by another process, such as being attached to an active Hyper-V VM it would be good for the process to fail at the start to avoid waiting 4 minutes to be told there is an issue. Please also make it clearer that there was an issue.

**Additional context**
![image](https://github.com/user-attachments/assets/29274d41-d63d-4d50-93c6-0326a9eada7e)

**Comment by OSDeploy** [comment]
True, this would be helpful, but no need wasting another 4 minutes.  Shut down the VM or disconnect the ISO and run New-OSDCloudISO and it will create the new ISOs in a few seconds.

**Comment by OSDeploy** [comment]
Closing this as there is no further effort going into the current OSDCloud.  The next release of OSDCloud will not possible to encounter this scenario as you will have to rebuild winpe into a new BootMedia, rather than editing over an existing BootMedia.
